swagger_spec_validator/util.py

```diff
@@ -48,6 +58,103 @@ def validate_spec_url(spec_url):
                        as `file://` this must be an absolute url for
                        cross-refs to work correctly.
     """
-    spec_json = load_json(spec_url)
+    spec_json = read_url(spec_url)
     validator = get_validator(spec_json, spec_url)
     validator.validate_spec(spec_json, spec_url)
+
+
+def _initialize_Swagger2dot0ObjectType(object_mapping_schema_path='schemas/v2.0/object_mappings.json'):
```

Comment:
What's the advantage of having these object mappings in a file (and allowing callers to specify another file)? If we expect library users to do that then this function should not be private. Otherwise it seems to me we could do this in a cleaner way by just writing out the enum directly...?